src/BassetManager.php

```diff
@@ -153,29 +163,39 @@ public function getUrl(string $asset): string
      */
     public function basset(string $asset, bool|string $output = true, array $attributes = []): StatusEnum
     {
+        $this->loading->start();
```

Comment:
Yup, there we go... this is a bit weird, right? Wouldn't it make more sense as:

```suggestion
        $this->loader->start();
```

Reply:
Or maybe:
```suggestion
        $this->loadingLog->start();
```

Something like that?